source/addComment.js

```diff
@@ -1,3 +1,25 @@
 $(document).ready(function(){
-  // add your code here
-})
+
+  $( "#new_comment_button" ).click(function() {
+    $("#new_comment").show();
+    $( "#new_comment_button" ).hide();
+  });
+
+  $( 'form#new_comment input[type="submit"]' ).click(function(event) {
+    event.preventDefault();
+    var textBoxCommentStuff = $( 'form#new_comment textarea' ).val();
+    var authorNameThing = $( 'form#new_comment input#authorName' ).val();
+
+    if (textBoxCommentStuff != "") {
```

Comment:
Would have been cool to tell the user that the field should be blank in some way i.e message, alert etc